tensorflow/compiler/xla/service/gpu/target_util.h

```diff
@@ -56,6 +56,9 @@ enum class TargetFunctionID {
   kFmod,
   kRound,
 };
+// AMDGCN target address spaces
+constexpr int kAMDGPUGlobalMemoryAddrSpace = 1;
+constexpr int kAMDGPUSharedMemoryAddrSpace = 3;
```

Comment:
@sumesh13 do you want to amend the PR so it covers `kNVPTXSharedMemoryAddrSpace` and `kAMDGPUSharedMemoryAddrSpace` as well?

Reply:
Thanks for pointing that out! I incorrectly assumed shared memory address is unused after looking for kAMDGPUSharedMemoryAddrSpace ...let me add that